src/main/java/com/powsybl/openloadflow/DefaultOuterLoopConfig.java

```diff
@@ -32,6 +32,7 @@ public List<OuterLoop> configure(LoadFlowParameters parameters, OpenLoadFlowPara
         if (!parameters.isNoGeneratorReactiveLimits()) {
             outerLoops.add(new ReactiveLimitsOuterLoop());
         }
+        outerLoops.add(new MonitoringVoltageOuterLoop());
```

Comment:
Should be conditioned by reactiveLimits params too ?

Reply:
I think there is no link with reactive limits as it is just if we have to switch once from a PQ bus to a PV bus. You can do that without any reactive limits. But after switching, the generator can be out of reactive limits...

Reply:
so we should add another parameters to be able to disable this outerloop